src/vdaf.rs

```diff
@@ -524,7 +524,7 @@ mod tests {
 
 pub mod poplar1;
 pub mod prg;
-#[cfg(feature = "prio2")]
+#[cfg(feature = "enpa")]
```

Comment:
Per David's comment, I think you'll want to put these back, because there are clients that want the Prio2 VDAF but don't want ENPA.

Reply:
I think the intent is that the following modules are behind the "enpa" feature:
* `util`
* `encrypt`
* `client`
* `server`

And the `vdaf::prio2` module is behind the "prio2" feature. This module in turn needs to compile the following structs from the "enpa" feature:
* `client::ClientMemory`
* `server::ValidationMemory`
* `server::VerificationMessage`
* `serverr::is_valid_share()`
* `util::proof_length`

So the question is how to resolve this triangular dependency. I see two ways:
1. Somehow make the "prio2" feature a superset of the "enpa" feature. There's likely a rusty way to do this but I wouldn't know how to spell it. (cc/ @tgeoghegan).
2. (My preferred solution) Move the functionality required by both into a new module and compile that module when either feature is in use.

Reply:
I think Chris' (2) is nicer in general, but (1) seems to be less work and achieves much the same thing. I took a stab at it in #270. In my branch, enabling feature `prio2` enables feature `enpa`, and then I also re-worked the GitHub actions so that we build, test, etc. under all supported feature combinations.

It'd be nice to use the `foo = ["dep:bar", "baz"]` formulation for defining features in `Cargo.toml`, so that you can disambiguiate between features and crate dependency names but I believe that requires Rust 1.60 or up, and I'm not sure it's worth bumping `prio`'s MSRV.